mwdb/model/migrations/versions/e3241d250d0c_add_group_admin_column_to_member_table.py

```diff
@@ -0,0 +1,28 @@
+"""Add group_admin column to member table
+
+Revision ID: e3241d250d0c
+Revises: e2d99ffcb8ce
+Create Date: 2020-09-17 15:24:00.140409
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'e3241d250d0c'
+down_revision = 'e2d99ffcb8ce'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('member', sa.Column('group_admin', sa.Boolean(), nullable=False))
```

Comment:
That would fail with non-empty `member` table:

Example: https://coderwall.com/p/8zw7bq/modifying-nullable-columns-in-alembic

You can just put raw SQL query in `execute` instead of using ORM:
```
op.execute("UPDATE \"member\" SET group_admin=TRUE WHERE group_admin IS NULL")
```